Foundation/NSSortDescriptor.swift

```diff
@@ -35,13 +35,13 @@ open class NSSortDescriptor: NSObject, NSSecureCoding, NSCopying {
     
     open var key: String? { NSUnimplemented() }
     open var ascending: Bool { NSUnimplemented() }
-    var keyPath: AnyKeyPath? { NSUnimplemented() }
+    public var keyPath: AnyKeyPath? { NSUnimplemented() }
```

Comment:
@ikesyo  I've made this public to match the implementation within the overlay as can be seen [here](https://github.com/apple/swift/blob/master/stdlib/public/SDK/Foundation/NSSortDescriptor.swift)